config/sync/field.field.config_pages.lockup_settings.su_upload_logo_image.yml

```diff
@@ -0,0 +1,38 @@
+uuid: 60392ecd-c02a-4c6d-9800-86400c6f279b
+langcode: en
+status: true
+dependencies:
+  config:
+    - config_pages.type.lockup_settings
+    - field.storage.config_pages.su_upload_logo_image
+  module:
+    - image
+id: config_pages.lockup_settings.su_upload_logo_image
+field_name: su_upload_logo_image
+entity_type: config_pages
+bundle: lockup_settings
+label: 'Upload logo image'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  file_directory: '[date:custom:Y]-[date:custom:m]'
+  file_extensions: 'png gif jpg jpeg'
+  max_filesize: ''
+  max_resolution: 1080x600
+  min_resolution: ''
+  alt_field: true
+  alt_field_required: true
```

Comment:
This alt text isn't getting used. In fact the alt text doesn't change at all and only uses the site name that is set in another config page. kinda confusing.

Reply:
I suppose we could just remove the alt altogether. I'd be ok with that.